graphics/tinygl/init.cpp

```diff
@@ -44,7 +44,15 @@ void initSharedState(GLContext *c) {
 void endSharedState(GLContext *c) {
 	GLSharedState *s = &c->shared_state;
 
-	free_texture(c, 0);
```

Comment:
And I forgot to say that I agree that close methods in general should only cause the de-allocation of disposed elements. I came to that conclusion when deciding to call `Graphics::Internal::tglCleanupImages` and forgot to go back and question my `endSharedState` change.